### PR TITLE
Identity tokens

### PR DESCRIPTION
**Pull request by jingming**

### CODE REVIEW

File: src/Twilio.Auth/AccessToken.cs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+        /// <summary>
+        /// Generate a JWT with the provided information and sign it with the given secret.
+        /// </summary>
+        /// <param name="secret">The secret to sign the token with</param>
+        /// <returns>Signed JWT authorizing the grants configured on this object</returns>
+        public string ToJWT()
+        {
+            var headers = new Dictionary<string, object>();
+            headers.Add("cty", "twilio-sat;v=2");
```
Comment (markandrus) [inline src/Twilio.Auth/AccessToken.cs]: per mario, content-type should be `twilio-fpa;v=1`

File: src/Twilio.Auth/Twilio.Auth.csproj [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="AccessToken.cs" />
+    <Compile Include="ConversationGrant.cs" />
```
Comment (senthgit) [inline src/Twilio.Auth/Twilio.Auth.csproj]: This needs to be ConversationsGrant.cs
Breaking the builds right now.

**Comment by senthgit** [comment]
@jingming looks like you will have to resolve conflicts

**Comment by senthgit** [comment]
+1

**Pull request merged**